src/test_insertion_sort.py

```diff
@@ -0,0 +1,62 @@
+"""Tests for Python implementation of a bubble sort sort."""
+import pytest
+from insertion_sort import insertion_sort as sort
```

Comment:
You shouldn't import as `sort`. this is already a built-in python method that you would be over riding.